cms/templates/course_outline.html

```diff
@@ -199,7 +199,8 @@ <h3 class="title-3">${_("Setting release dates and grading policies")}</h3>
                 <p>${_("Select the Configure icon for a section or subsection to set its release date. When you configure a subsection, you can also set the grading policy and due date.")}</p>
                 <h3 class="title-3">${_("Changing the content students see")}</h3>
```

Comment:
@efischer19  Would it be out of scope to change "students" to "learners" here? :-)